### ISSUE TITLE
configuration of mailer email for self hosted version

### ISSUE THREAD

**Issue by Joshua-Flores** [plausible/docs#53]
I've updated my .env file with a custom email address, but the emails coming from plausible still show as from hello@plausible.local. Am I missing a configuration step here?

**Comment by ukutaht** [comment]
Difficult to help without more info. You should configure `MAILER_EMAIL=my@email.com` to change the mailer email. Could you paste your config so I can attempt to reproduce?

**Comment by Joshua-Flores** [comment]
Hey Uku, I actually got it working. What I did wrong was I had the email address listed as 'email@subdomain.mywebsite.com' which is why it wasn't working and defaulting to local. I removed the subdomain from the MAILER_EMAIL and it worked. 

Side question now - I'm getting a flag in gmail saying:

_"Gmail couldn't verify that 'mywebsite.com' actually sent this message (not a spammer)."_ 

Here's my conf file as it exists right now. I'm wondering if I'm missing something related to mail.

ADMIN_USER_EMAIL=me@gmail.com
ADMIN_USER_NAME=NAME
ADMIN_USER_PWD=PASSWORD
BASE_URL=https://plausible.mywebsite.com
SECRET_KEY_BASE=SECRETKEY
DISABLE_REGISTRATION=true
MAILER_EMAIL=email@mywebsite.com

I appreciate your help!
